### PR TITLE
Add syntax for multiple selector logging

### PR DESCRIPTION
**Pull request by AndyHunt66**
Add syntax for multiple selector logging

### CODE REVIEW

**Comment by andresrc** [comment]
@kvch @urso is the syntax correct?

**Comment by urso** [comment]
Yes, but selectors is just an array. YAML arrays can be written using `[]` or `-` e.g.

```
logging.selectors: [a, b]
```

vs

```
logging.selectors:
- a
- b
```

**Comment by urso** [comment]
> Should this update include both the YAML array formats you have listed above?

Not necessarily. But instead of we mention "following syntax", maybe we want to say "YAML list syntax" and link to our configuration file format docs.

File: libbeat/docs/loggingconfig.asciidoc [hunk 1]
```diff
@@ -156,6 +156,12 @@ To see which selectors are available, run {beatname_uc} in debug mode
 after the log level and is enclosed in brackets.
 =====
 
+To configure multiple selectors, use the following syntax:
```
Comment (EamonnTP) [inline libbeat/docs/loggingconfig.asciidoc:159]:
```suggestion
To configure multiple selectors, use the following {beats-ref}/config-file-format.html[YAML list syntax]:
```

**Review by EamonnTP — CHANGES_REQUESTED** [review]
@AndyHunt66 Thanks for the update! Just a minor suggestion.

**Review by EamonnTP — APPROVED** [review]

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**